README.md

```diff
@@ -54,7 +54,9 @@ node server
 ```
 
 The defaults without a config file should work for a local Cassandra
-installation with the default passwords. To customize RESTBase's behavior,
+installation with the default passwords. Restbase has been tested with
+[Cassandra 2.1.13](http://wiki.apache.org/cassandra/DebianPackaging).
```

Comment:
It's the link to the installation guide and might help people to install cassandra. It says
> You will want to replace 21x by the series you want to use: 20x for the 2.0.x series, 12x for the 1.2.x series, etc... You will not automatically get major version updates unless you change the series, but that is a feature.